src/attributes.jl

```diff
@@ -776,13 +776,17 @@ ConstraintDual() = ConstraintDual(1)
 """
     ConstraintBasisStatus()
 
-A constraint attribute for the `BasisStatusCode` of some constraint, with
-respect to an available optimal solution basis.
+A constraint attribute for the `BasisStatusCode` of some constraint in result
+`N`, with respect to an available optimal solution basis. If `N` is omitted, it
+is 1 by default.
 
 **For the basis status of a variable, query the corresponding `SingleVariable`
 constraint that enforces the variable's bounds.**
 """
-struct ConstraintBasisStatus <: AbstractConstraintAttribute end
+struct ConstraintBasisStatus <: AbstractConstraintAttribute
+    N::Int
```

Comment:
This doesn't match the naming conventions for fields. What about `result_index`?

Reply:
It's `N` for `VariablePrimal`, `ConstraintPrimal` and `ConstraintDual` but `result_index` for `ObjectiveValue` and `DualObjectiveValue`. `result_index` is indeed better.